R/learner_stats_regr_glm.R

```diff
@@ -0,0 +1,116 @@
+#' @title Regression Generalized Linear Model Regression Learner
+#' @author salauer
+#' @name mlr_learners_regr.glm
+#'
+#' @template class_learner
+#' @templateVar id regr.glm
+#' @templateVar caller glm
+#'
+#'
+#' @template seealso_learner
+#' @template example
+#'
```

Comment:
Can you please add:

"@details For logistic regression please use `classif.log_reg`."

Reply:
I had a question about that. According to NEWS.Rd:
```
# mlr3extralearners 0.1.0

- Initial release. mlr3extralearners contains all learners from the mlr3learners organisation, which is now archived.
```
but I don't see that included in this package. Similar to Poisson regression it could be done through `glmnet` with `lambda=0`. Should I create a `classif.glm` that could also allow for `family="quasibinomial"`?

Reply:
The mlr3learners organisation is not the same as the mlr-org organisation or the `mlr3learners` repo (though the naming was confusing). `classif.log_reg` lives in the repo `mlr3learners`